indico/web/templates/forms/multiple_tag_select_widget.html

```diff
@@ -9,6 +9,10 @@
         setupMultipleTagSelectWidget({
             fieldId: {{ field.id | tojson }},
             choices: {{ field.choices | tojson }},
+            inputArgs: {{ input_args | tojson }},
+            initialSelection: {{ field.data | default([]) | tojson }},
         });
+        {# Call `initForms()` again because `WTFMultipleTagSelectField` has been rendered only after its first call. #}
+        initForms($('form'));
```

Comment:
That sounds kind of messy, but I don't know  if there's a better solution for it TBH.

Do you have any idea why the field only gets rendered after this is called?